ui/page/wallet_page.go

```diff
@@ -169,9 +179,14 @@ func (pg *WalletPage) OnResume() {
 			listItem.backupAcctClickable = new(widget.Clickable)
 			listItem.collapsible = pg.Theme.CollapsibleWithOption()
 		}
-
-		pg.listItems = append(pg.listItems, listItem)
+		pg.listLock.Lock()
+		listItems = append(listItems, listItem)
+		pg.listLock.Unlock()
 	}
+
+	pg.listLock.Lock()
+	pg.listItems = append(pg.listItems, listItems...)
```

Comment:
Is append the intended operation here?
```suggestion
	pg.listItems = listItems
```